diff --git a/dist/core/index.mjs b/dist/core/index.mjs
index 7b0074a58e7dc780c6e7f9240e1874e89454d9d2..b00f9f0fa9dfe4d8bcd2a74c2f882aaa17bb8212 100644
--- a/dist/core/index.mjs
+++ b/dist/core/index.mjs
@@ -2102,7 +2102,11 @@ async function prerender(nitro) {
       {
         headers: { "x-nitro-prerender": encodedRoute },
         retry: nitro.options.prerender.retry,
-        retryDelay: nitro.options.prerender.retryDelay
+        retryDelay: nitro.options.prerender.retryDelay,
+        async onResponseError({response}) {
+          nitro.logger.warn(`Error ${response.status} fetching route: ${route}, 
+response body: ${await response.text()}`);
+        }
       }
     );
     let dataBuff = Buffer.from(await res.arrayBuffer());
@@ -2201,6 +2205,9 @@ async function prerender(nitro) {
       parents?.size ? `
 ${[...parents.values()].map((link) => colors.gray(`  \u2502 \u2514\u2500\u2500 Linked from ${link}`)).join("\n")}` : "";
       nitro.logger.log(formatPrerenderRoute(route));
+      if (route.error) {
+        nitro.logger.error(route.error);
+      }
     }
     nitro.logger.log("");
     throw new Error("Exiting due to prerender errors.");
